build/wikia/transformers/transformWarframe.js

```diff
@@ -3,14 +3,7 @@
 const getColors = require('get-image-colors')
 const imageDownloader = require('image-downloader')
 
-const POLARITIES = {
-  Bar: 'naramon',
-  V: 'madurai',
-  D: 'vazarin',
-  U: 'umbra',
-  Ability: 'zenurik',
-  R: 'unairu'
-}
+const POLARITIES = require('../../../config/wikiPolarities')
```

Comment:
I was going to use these for mods, but since they're on our data already, i didn't think it would be useful. I can revert this.